helm/alfresco-content-services/esc_values.yaml

```diff
@@ -0,0 +1,25 @@
+alfresco-elasticsearch-connector:
+  enabled: true
+  reindexing:
+    enabled: true
+
+repository:
+  replicaCount: 1
+  extraLogStatements:
```

Comment:
This was only for test purposes, I'm going to remove it during the first commit after the pull request review.